Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-@Library(value='iow-ecs-pipeline@2.2.0', changelog=false) _
+@Library(value='iow-ecs-pipeline@3.0.0', changelog=false) _
```

Comment:
Does this exist?  I assume yes, but last I looked it was 2.2.4 or something.

Reply:
Yeah this is new as of really recently, started with IOW-785 and then some breaking changes for ecs deploys were introduced with the removal of autoscaling logic for manually managed ec2 instances either late last week or early this week, hence the new major version tag.